mumbleBot.py

```diff
@@ -648,16 +718,13 @@ def clear(self):
         # Kill the ffmpeg thread and empty the playlist
         self.interrupt()
         var.playlist.clear()
-        self.wait_for_ready = False
         self.log.info("bot: music stopped. playlist trashed.")
 
     def stop(self):
         self.interrupt()
         self.is_pause = True
-        if len(var.playlist) > 0:
```

Comment:
Can you explain this part?

Logics related to the loop and its state are messy and modifications, if not thoroughly tested, if prone to errors. Are you sure this will work?

Reply:
As you can see, this part was introduced in https://github.com/azlux/botamusique/commit/fceb50e8943fb1f92537cd59b3b2ed16c14ce29b, to fix a bug. I think you need to keep the part unchanged.